lib/compile.js

```diff
@@ -105,6 +106,26 @@ function compileLoad(loader, load, compileOpts, cache) {
     if (format == 'cjs')
       mappedLoad.source = mappedLoad.source.replace(hashBangRegEx, '');
     return Promise.resolve()
+    .then(function () {
+      var metadata = mappedLoad.metadata;
+      if (metadata.sourceMap)
+        return;
+      var sourceMap = sourceMappingURL.getFrom(mappedLoad.source);
+      if (!sourceMap)
+        return;
+      mappedLoad.source = sourceMappingURL.removeFrom(mappedLoad.source);
+      if (sourceMap.startsWith('data:')) {
+        // inline source map
+        metadata.sourceMap = JSON.parse(dataUriToBuffer(sourceMap).toString('utf8'));
```

Comment:
Sure, if you like we can leave it as a full thrown error for now, and rather handle this in future when it comes up again.